src/libraries/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -110,11 +111,11 @@ public void TimesIncludeMillisecondPart()
             });
         }
 
-        [ConditionalFact(nameof(isHFS))]
-        public void TimesIncludeMillisecondPart_HFS()
+        [ConditionalFact(nameof(WithMillisecondResolution))]
+        public void TimesIncludeMillisecondPart_HFSOrBrowser()
```

Comment:
Can we change the name to be more general in case more platforms are added that also do not support fractions of seconds resolution?